### DISCUSSION TITLE
Github Copilot: Unable to display suggestions in VSCode despite solutions between synthesized

### DISCUSSION THREAD

**Discussion by CharlesBordet** [community/community#44856] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

I activated my 60-day trial of Github Copilot yesterday, and so far I haven't been able to display any code suggestion. 

### Code example

I have tried the example from the documentation:

* Create new file, save it as `test.js`
* Start defining a function `function calculateDaysBetweenDates(begin, end) {`
* Nothing happens.

### Diagnostics

If I run the command **Github Copilot: Collect Diagnostics**, I get this page:

```
## Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Extensions

- Copilot Version: 1.69.8032
- Is `win-ca` installed?: true
- Is `mac-ca` installed?: true

## Node setup

- Number of root certificates: 131
- Operating system: Linux
- Operating system version: 5.15.0-58-generic
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 
```

Everything seems fine.

I am running:

* Ubuntu 22.04 LTS
* VSCode 1.74.3
* GitHub Copilot extension v1.69.8032

I am not behind a VPN or a proxy.

### What I tried

If I go to **View / Output** and choose GitHub Copilot from the dropdown menu, nothing is displayed.

On the bottom status bar, right-side, there is an icon that hovers the message "Deactivate Copilot". If I click, I have the option to disable it globally and disable it for javascript. This tends to indicate the current status is enabled.

In Settings, the Github > Copilot option is set to "Enable". The Github > Copilot > Inline Suggest is set to "Enable" and the checkbox next to "Show inline suggestions" is checked.

I tried uninstalling/re-installing VSCode multiple times. To install it, I download the deb file and run `sudo gdebi code.deb` in a terminal.

I tried uninstalling/re-installing the Github Copilot extension multiple times. I tried installing the version before the latest one as well. I closed and re-opened VSCode between these attempts.

I tried signing in and out of Github multiple times, but it didn't change anything. I also closed and re-opened VSCode.

I tried running the command "Open Github Copilot" and I can see some code suggestions, so the connection to Github seems to work. Below is the first few lines of these code suggestions:

```
Synthesizing 9/10 solutions

=======

function calculateDaysBetweenDates(begin, end) {
  const date1 = new Date(begin);
  const date2 = new Date(end);
  const diffTime = Math.abs(date2 - date1);
  const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
  return diffDays;
}

=======

function calculateDaysBetweenDates(begin, end) {
  const beginDate = new Date(begin);
  const endDate = new Date(end);
  const difference = endDate.getTime() - beginDate.getTime();
  return Math.floor(difference / (1000 * 60 * 60 * 24));
}

=======

function calculateDaysBetweenDates(begin, end) {
  const beginDate = new Date(begin);
...
```

They're not displayed here, but there are small links "Accept Solution" above each solution that I can click on and it will write it on my script. This is not convenient at all however.

In the Github settings of my account, the option "Suggestions matching public code" is set to **Allow**. The checkbox "Allow GitHub to use my code snippets for product improvements *" is unchecked.

What else can I try? I feel like I'm running out of options.

**Comment by FlorianEmpinet** [comment]
Same problem here.. but it works on PyCharm..
